src/interaction/pointer.ts

```diff
@@ -12,16 +12,16 @@ let timeout: number = null;
 
 export function start(): void {
     reset();
-    bind(document, "mousedown", mouse.bind(this, Event.MouseDown));
-    bind(document, "mouseup", mouse.bind(this, Event.MouseUp));
-    bind(document, "mousemove", mouse.bind(this, Event.MouseMove));
-    bind(document, "mousewheel", mouse.bind(this, Event.MouseWheel));
-    bind(document, "dblclick", mouse.bind(this, Event.DoubleClick));
-    bind(document, "click", mouse.bind(this, Event.Click));
-    bind(document, "touchstart", touch.bind(this, Event.TouchStart));
-    bind(document, "touchend", touch.bind(this, Event.TouchEnd));
-    bind(document, "touchmove", touch.bind(this, Event.TouchMove));
-    bind(document, "touchcancel", touch.bind(this, Event.TouchCancel));
+    bind(document, "mousedown", mouse.bind(this, Event.MouseDown), true);
```

Comment:
What's the true mean? Add comment.

Reply:
It's defined in Typescript schema (capture the event).

Reply:
Maybe but it's surprisingly hard to find documentation about this -- e.g., the bind page on MDN doesn't make it clear. What does "capture the event" mean? Is this "addEventListener" (which has clearer documentation and seems to be what you are doing here).  
[https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener)

It would be clearer if you worte this as:

document.addEventListender((, <event, mouse.bind(....), {capture: true})

Boiolean optoins are too opauqe and should be avoided. 

BTW, should this not also add teh "passive:true" optimization to help with scrolling, etc. 


---
In reply to: [334240802](https://github.com/microsoft/clarity-js/pull/151#discussion_r334240802) [](ancestors = 334240802)